webapp/src/components/SendMessage.js

```diff
@@ -11,9 +11,17 @@ class SendMessage extends React.Component {
   };
 
   onSubmit = async () => {
-    const response = this.state;
+    await fetch(`/api/channels/${this.state.channelId}/messages`, {
```

Comment:
Utiliser `this.props.channelId` (et donc pas besoin de déclarer `channelId` dans le `state`).